packages/metal-quartz-icon/src/MetalQuartzIcon.js

```diff
@@ -20,25 +19,33 @@ class MetalQuartzIcon extends Component {
  */
 MetalQuartzIcon.STATE = {
 	/**
-	 * Path to the spritemap file with the icons.
+	 * CSS classes to be applied to the element, use this to pass in classes for
+	 * third-party icon libraries e.g. `glyphicon glyphicon-plus`. If
+	 * elementClasses is defined and spritemap or symbol is undefined, it will
+	 * insert elementClasses into a span tag.
+	 * @type {string}
+	 */
+	elementClasses: {
+	},
+
+	/**
+	 * The path to the SVG spritemap file containing the icons.
 	 * @instance
 	 * @memberof MetalQuartzIcon
 	 * @type {string}
 	 */
 	spritemap: {
-		required: true,
-		validator: core.isString
+		validator: value => core.isString(value)
```

Comment:
It should work fine with just `validator: core.isString`, that's what we do in a lot of metal components. Did this result in different behavior?